modules/tide_workflow_notification/tide_workflow_notification.install

```diff
@@ -15,3 +15,13 @@ function tide_workflow_notification_install() {
     user_role_grant_permissions($role_name, $permissions);
   }
 }
+
+/**
+ * Implements hook_update_N().
+ */
+function tide_workflow_notification_update_8001() {
```

Comment:
Shouldn't you enable the module here too?

Reply:
ah. yes. Thanks @anthony-malkoun

Reply:
Is `tide_workflow_notification` installed anywhere yet? Could this just go in the install hook?

Reply:
yes. @GROwen  thanks.